flink-ai-flow/ai_flow/api/ops.py

```diff
@@ -469,18 +512,18 @@ def action_on_model_version_event(job_name: Text,
                                   model_name: Text,
                                   model_version_event_type: Text,
                                   namespace: Text = DEFAULT_NAMESPACE,
-                                  action: JobAction = JobAction.RESTART,
-                                  ) -> None:
+                                  action: JobAction = JobAction.RESTART):
     """
-    Add model version control dependency. It means src channel will start when and only a new model version of the
-    specific model is updated in notification service.
-
-    :param action: ai_flow.workflow.control_edge.TaskAction
-    :param namespace: the namespace of the event.
-    :param model_version_event_type: one of ai_flow.model_center.entity.model_version_stage.ModelVersionEventType
-    :param job_name: The job name
-    :param model_name: Name of the model, refers to a specific model.
-    :return: None.
+    Add model version control dependency. It means the source channel will start when and only a new model version of
```

Comment:
From my understanding, in our refractored API, we do not have `channel` concept in the `action_on_xx` API. Instead, we use the `job_name`. 

If we agree on this point, I think it is better to change `It means the source channel will start when and only a new model version of the specified model is updated in notification service.`  to `It means the job will take given action when the specified model is updated to given model_version_event_type. `